policy/action.go

```diff
@@ -372,7 +374,9 @@ func createActionConditionKeyMap() ActionConditionKeyMap {
 				condition.ExistingObjectTag.ToKey(),
 			}, commonKeys...)...),
 
-		HeadBucketAction: condition.NewKeySet(commonKeys...),
+		HeadBucketAction:                 condition.NewKeySet(commonKeys...),
+		GetObjectAttributesAction:        condition.NewKeySet(commonKeys...),
```

Comment:
https://docs.aws.amazon.com/AmazonS3/latest/userguide/list_amazons3.html Actually this supports tags as well so you need to add 

```
		GetObjectAttributesAction: condition.NewKeySet(
			append([]condition.Key{
				condition.S3VersionID.ToKey(),
				condition.ExistingObjectTag.ToKey(),
			}, commonKeys...)...),
```

Not just commonKeys.

Reply:
GetObjectAttributes did not require the versionID, the GetObjectVersionAttributes was the one. 

But they've been added now.